opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchScrollRequest.java

```diff
@@ -34,32 +38,38 @@
 @Getter
 @ToString
 public class OpenSearchScrollRequest implements OpenSearchRequest {
-
+  private SearchRequest initialSearchRequest;
   /** Scroll context timeout. */
-  private final TimeValue scrollTimeout;
+  private TimeValue scrollTimeout;
 
   /**
    * {@link OpenSearchRequest.IndexName}.
    */
-  private final IndexName indexName;
+  private IndexName indexName;
 
   /** Index name. */
   @EqualsAndHashCode.Exclude
   @ToString.Exclude
-  private final OpenSearchExprValueFactory exprValueFactory;
-
+  private OpenSearchExprValueFactory exprValueFactory;
   /**
    * Scroll id which is set after first request issued. Because ElasticsearchClient is shared by
    * multi-thread so this state has to be maintained here.
    */
   @Setter
   @Getter
-  private String scrollId;
+  private String scrollId = NO_SCROLL_ID;
+
+  public static final String NO_SCROLL_ID = "";
 
   private boolean needClean = false;
 
-  /** Search request source builder. */
-  private final SearchSourceBuilder sourceBuilder;
+  @Getter
+  private List<String> includes;
+
+  /** Default constructor for Externalizable only.
+   */
+  public OpenSearchScrollRequest() {
```

Comment:
Make `@Deprecated`

Reply:
I see it is not used anymore in deserialization. This class doesn't implement `Readable` or `Externalizable`.
Can you remove this constructor?

Reply:
Good catch.